test/onnx/test_pytorch_onnx_caffe2.py

```diff
@@ -1019,6 +1019,15 @@ def forward(self, x):
         x = torch.ones(3, 3)
         self.run_model_test(AddMMModel(), input=x, train=False, batch_size=BATCH_SIZE)
 
+        class CatModel(torch.nn.Module):
```

Comment:
We generally don't add tests for new functionality to Caffe2 since we don't really care about it except to avoid regressions.